packages/blip/app/components/chart/index.d.ts

```diff
@@ -26,82 +26,17 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-import MedicalDataService, { Datum } from 'medical-domain'
-import { utils as vizUtils } from 'tidepool-viz'
-import DialogDatePicker from '../../../../yourloops/components/date-pickers/dialog-date-picker'
+import MedicalDataService, { Datum, TimePrefs } from 'medical-domain'
+import DataUtil from 'tidepool-viz/src/utils/data'
```

Comment:
Weird import... It is usually only `'tidepool-viz'`. Isn't your component exported by viz?